src/test/resources/GET.feature

```diff
@@ -1,15 +1,24 @@
+
 @Get
 
-Feature: Get users information
+Feature: Get the information of all registered users
 
   Background:
     Given I am working on "QA" environment
     And I am targeting "at-sce-api" service
+    And I have access to "at-sce-db" database
 
-
-@US_001 @1
-  Scenario: Get all users
+  @1
+  Scenario: Get all registered users in the DB with the correct information when the number of users is > 0
   Given I want to retrieve all users
   When I send a GET request
   Then the status code should be "200"
-  And information retrieved from service should match with DB
\ No newline at end of file
+  And the number of registered users should be the same of the number of retrieved users
+  And information retrieved from service should match with DB
+  @2
+  Scenario: Get an empty list when there aren't registered users in the DB
```

Comment:
How are you going to validate this scenario?

Reply:
When I tested this scenario, I erased all users of the data base manually.
I have the reports of this scenario.